catalog/ui/catalog-ui-search/src/main/webapp/component/search-form/search-form-loading.view.js

```diff
@@ -0,0 +1,28 @@
+/**
```

Comment:
Unless we're going to do something fancy with the loading, we can probably put the spinner directly in the view that needs it (and hide and show it there).

Reply:
done